host_guest/Reference/README.md

```diff
@@ -0,0 +1,12 @@
+# Main SAMPL challenge reference calculation files
+
+These input files will be used to run the reference calculations for the main SAMPL challenge. More details about the exact methods will be provided later. Files are available in Amber (`prmtop`/`rst7`), Gromacs (`top`/`gro`), OpenMM (`XML`) and PDB formats. Solvated system files are available both for the host-guest complex (e.g. `complex`) and the guest alone (e.g. `solvent`).
```

Comment:
I would think you'd want to link to this from where you first mention the reference calculations in the `host_guest/README.md` to make it easy for people to get here, and say just a LITTLE more here -- specifically, that you're running reference calculations (as organizers have in previous SAMPL challenges) with Yank, and that these are intended to provide a point of comparison across all targets for other methods. This is distinct from the SAMPLing challenge, where we are explicitly asking people to use the exact same inputs and details and compare EXCLUSIVELY sampling efficiency. For reference calculations, you're just making available inputs for the calculations you plan to run in case anyone would like to run the same systems to compare performance.